docs/sources.md

```diff
@@ -130,6 +130,10 @@ This looks like:
     // https://backstage.io/docs/auth/service-to-service-auth/#usage-in-external-callers
     token: '$(BACKSTAGE_TOKEN)',
 
+    // Optional. Customize header value used for authorization. If you are using a custom 
+    // value for your authorization header that can be changed here.
+    header: 'CustomHeader'
```

Comment:
I suspect a majority of people won't want to override the header — so I think it'd be good to comment this out, as if folks copy/paste this config, it'll fail:

```suggestion
    // By default, we send the token in the `Authorization: Bearer your_token` HTTP
    // header. If you want to override this, then you use this optional configuration
    // value to override the header name that is used.
    // header: 'CustomHeader'
```

Once that's done, I'm happy to get this merged and released for you!

Reply:
Good point @samstarling, I haven't taken that into consideration initially.